uefi/filesection.go

```diff
@@ -37,9 +37,10 @@ type FileSection struct {
 
 	//Metadata for extraction and recovery
 	ExtractPath string
+	fileOrder   int
 }
 
-// Assemble assembles the Firmware File from the binary
+// Assemble assembles the section from the binary
```

Comment:
Why doesn't assemble read the `fileOrder`? How does it get the sections back into the original order?

Reply:
Never mind, I see that the information is stored in the JSON.